Sources/Shared/Core/HorizontalBlueprintLayout.swift

```diff
@@ -147,12 +148,12 @@
 
           if stickyHeaders {
             headerAttribute?.frame.origin.x = headerFooterX
-            headerAttribute?.frame.size.width = min(headerFooterWidth, widthOfSection)
+            headerAttribute?.frame.size.width = max(headerFooterWidth, widthOfSection)
```

Comment:
@christoff-1992 you reckon that this change makes sense?
It does in my head... that the largest of the two should be used. The header should always stretch to the end of the section right? Or perhaps it shouldn't now that I think of it. Perhaps it should actually be `min` here if you are using sticky headers. Then it should simply move across the screen and not span the entire section.

Reply:
I reverted this change and made some updates in 
https://github.com/zenangst/Blueprints/pull/58/commits/2148166afec3029905c19709c9211ebff9615fe0